fastestimator/dataset/op_dataset.py

```diff
@@ -15,29 +15,39 @@
 from copy import deepcopy
 from typing import List
 
+import numpy as np
 from torch.utils.data import Dataset
 
-from fastestimator.dataset.unpaired_dataset import UnpairedDataset
-from fastestimator.op import NumpyOp, get_inputs_by_op, write_outputs_by_op
+from fastestimator.dataset.batch_dataset import BatchDataset
+from fastestimator.op.op import NumpyOp, get_inputs_by_op, write_outputs_by_op
```

Comment:
why are we using fe.op.op for NumpyOp but using fe.util rather than fe.util.util?

Reply:
changed everything to folder import